@navikt/core/react/src/table/stories/people.json

```diff
@@ -0,0 +1,822 @@
+[
```

Comment:
Bare litt småpirk, men kan dette autogenereres med eks (nye) Faker? https://www.npmjs.com/package/@faker-js/faker

Reply:
Om dette skal være en Chromatic-story får vi da trøbbel med ustabil data. Eller kan man kanskje bruke Faker på en deterministisk måte?

Reply:
Ah ja sant det. Fant dette fra nevnt faker-pakke over
> Setting a randomness seed
If you want consistent results, you can set your own seed:
```
faker.seed(123);

const firstRandom = faker.datatype.number();

// Setting the seed again resets the sequence.
faker.seed(123);

const secondRandom = faker.datatype.number();

console.log(firstRandom === secondRandom);
```

Reply:
Se der, ja. Kult!

Jeg er enig i at stories her kunne vært ryddigere, men jeg prioriterer ikke å fikse det nå.